.travis.yml

```diff
@@ -143,3 +149,10 @@ jobs:
             - node
       before_install:
         - export JAVA_HOME="$(/usr/libexec/java_home -v 1.8)"
+        - pip install PyGithub  # used to upload packages to Github releases
+      script:
+        - if [[ -z "${TRAVIS_TAG}" ]]; then python src/packaging/set_commit_and_date_in_version.py $TRAVIS_COMMIT; fi
+        - make distrib -j2
+# The following lines are commented for now because they make the test fail because the creation of the package sometimes faile on mac
```

Comment:
```suggestion
# The following lines are commented for now because they make the test fail because the creation of the package sometimes fails on mac
```